derive/src/utils.rs

```diff
@@ -136,6 +159,11 @@ pub fn canonical(name: &syn::Ident, method_sig: &syn::MethodSig) -> String {
 	s
 }
 
+/// Returns the Keccak hash (256-bits) of the given string slice.
+/// 
+/// # Note
+/// 
+/// The given string slice will be interpreted as bytes array.
```

Comment:
As such, maybe it'd be better if this function actually took the byte slice?

Reply:
Good point! Did so in the latest set of commits.